src/main/resources/mybatis/lookup.xml

```diff
@@ -78,7 +78,7 @@
         select featureid
           from nhdplus.catchmentsp
             where st_intersects(nhdplus.catchmentsp.the_geom,
-              st_setsrid(st_makepoint(#{lon}, #{lat}), 4269)::geography);
+              st_geomfromtext('POINT(${lon} ${lat})', 4269));
```

Comment:
This might be out of scope for this ticket/change. Are there tests that check if the featureid returned by
the query has the value expected? If not, probably worth at least creating a ticket.

Reply:
Yes, this test already exists in NetworkControllerPositionIT.  It doesn't show up in this pull request because the pull request is just a refinement to get the index being used.